src/acl/AnnotateTransaction.h

```diff
@@ -6,8 +6,8 @@
  * Please see the COPYING and CONTRIBUTORS files for details.
  */
 
-#ifndef SQUID_ACLANNOTATETRANSACTION
-#define SQUID_ACLANNOTATETRANSACTION
+#ifndef SQUID_ACLANNOTATETRANSACTION_H
+#define SQUID_ACLANNOTATETRANSACTION_H
```

Comment:
It would be best to add the new macro syntax to avoid unnecessary double-touching and adjust the `#endif` as well for clarity that this was not broken earlier.
```suggestion
#ifndef SQUID_SRC_ACL_ANNOTATETRANSACTION_H
#define SQUID_SRC_ACL_ANNOTATETRANSACTION_H
```